scheduler/servicestatemanager/servicestatemanager_unit_test.go

```diff
@@ -2025,6 +2024,8 @@ func (s *ServiceStateManagerSuite) TestServiceStateManager_queueLoop(c *C) {
 	done := make(chan struct{})
 	go func() {
 		wg.Wait()
+		// Wait for all services to get scheduled so we don't randomly fail assertExpectations below
+		s.serviceStateManager.WaitScheduled("tenant1", "A", "D", "G", "H")
```

Comment:
Is this test function testing anything that TestServiceStateManager_WaitScheduled doesn't now?